### DISCUSSION TITLE
importing aliases

### DISCUSSION THREAD

**Discussion by KrystianPlichta** [modoboa/modoboa#2436] [Q&A]
I have a problem, when importing a csv file with aliases I get an error:
modoboa.lib.exceptions.NotFound Local recipient: randmail@gmail.com has not been found

On the previous postfix server when a user quits we disable the user's mail account but create an alias leading to an external mail. In this case, modoboa wants the user the alias points to to be on the modoboa server.
Any idea how to solve this?

**Comment by tonioo** [comment]
Has the message suggests, the address you specify is not a local recipient so it can't work. Can you give more detail?

**Reply by nealpaze** [reply]
This used to work in older versions of Modoboa. I was installing a fresh new version on a fresh new server and was trying to set up aliases and cannot seem to get this to work. The old system certainly allowed this which was very helpful. Is this no longer allowed, having an email alias that forwards to an external domain email address?
